### PR TITLE
Add delay for switch on/off on Fibaro double switch

### PR DESCRIPTION
**Pull request by ctowns**
https://smartthings.atlassian.net/browse/HCS-4161

### CODE REVIEW

File: drivers/SmartThings/zwave-switch/src/fibaro-double-switch/init.lua [hunk 1]
```diff
@@ -32,6 +32,9 @@ local ENDPOINTS = {
   child = 2
 }
 
+-- This value is carried over from the DTH
+local ON_OFF_DELAY_SEC = 5
```
Comment (greens) [inline drivers/SmartThings/zwave-switch/src/fibaro-double-switch/init.lua:36]: you could use `zwave.constants.MIN_DIMMING_GET_STATUS_DELAY`

File: drivers/SmartThings/zwave-switch/src/fibaro-double-switch/init.lua [hunk 2]
```diff
@@ -126,6 +129,19 @@ local function switch_report(driver, device, cmd)
...
 
+local function set_switch(value)
+  return function(driver, device, cmd)
+    local delay = ON_OFF_DELAY_SEC
+    local query_device = function()
+      local component = cmd and cmd.component and cmd.component or "main"
+      device:send_to_component(SwitchBinary:Get({}), component)
+    end
+
+    device:send_to_component(SwitchBinary:Set({ target_value = value, duration = 0}), cmd.component)
```
Comment (greens) [inline drivers/SmartThings/zwave-switch/src/fibaro-double-switch/init.lua:140]: DTH uses `Basic` for the Set, and Binary for the `Get`.

File: drivers/SmartThings/zwave-switch/src/fibaro-double-switch/init.lua (hunk 1 above)
Comment (ctowns) [reply drivers/SmartThings/zwave-switch/src/fibaro-double-switch/init.lua]: Done

File: drivers/SmartThings/zwave-switch/src/fibaro-double-switch/init.lua (hunk 2 above)
Comment (ctowns) [reply drivers/SmartThings/zwave-switch/src/fibaro-double-switch/init.lua]: Switch to using `Basic`

**Comment by ctowns** [comment]
Camille tested this and it is now working with these changes!

Also, I updated the on/off command to use `Basic:Set` as you said and I adjusted the test cases accordingly

**Review by greens — APPROVED** [review]

**Pull request merged**